packages/cf-component-text/src/Text.js

```diff
@@ -27,7 +27,7 @@ class Text extends React.Component {
 }
 
 Text.propTypes = {
-  size: PropTypes.oneOf(['normal', 'small']),
+  size: PropTypes.oneOf(['normal', 'small', 'bold']),
```

Comment:
I think it would be better to add a new prop `style` or `weight` and split `fontSize` and `fontWeight` definitions. It's weird to have `bold` size.